tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -66,14 +66,26 @@ then
   ${PIP} install grpcio --no-index --find-links "file://$ARTIFACT_DIR/"
   ${PIP} install grpcio-tools --no-index --find-links "file://$ARTIFACT_DIR/"
 
+  HEALTH_PROTO="src/proto/grpc/health/v1/health.proto"
```

Comment:
This ```GRPC_BUILD_GRPCIO_TOOLS_DEPENDENTS``` isn't always set, so on some platforms these might not get built.

Reply:
That is intentional—and this behavior is unchanged from the status quo—since we only have one distributed package for all platforms for reflection and health checking.

Reply:
Makes sense

Reply:
Worth adding a comment to the source?